### PR TITLE
added dboracle container 12c REL1

### PR DESCRIPTION
**Pull request by kurtz81**
added project to create a container with Oracle database 12c relase 1

### CODE REVIEW

File: envs/oracledb/12cr1/groups/12cr1.yaml [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+            echo "COMMON OPERATION" &&
+            echo "################"
+          - test -d ${oracle_base}"/"${db_version}"/" && (rm -rf /u01; echo "empty oracle dirs"; exit 0) || echo "oracle dirs not present"
+          - a=`yum repolist | grep epel`; test -n "$a" || yum install -y ${epel_repo} 
+          - yum install -y wget htop rlwrap sudo
+          - wget ${ol7_repo_prefix}"/"${ol7_repo}
+          - mv ${ol7_repo} /etc/yum.repos.d/          
+          - yum repolist
+          - echo "${motdfile}" > /etc/motd
+          - sed -i -e "s:HOSTNAME:${hostnamevar}:g" /etc/motd
```
Comment (geaaru) [inline envs/oracledb/12cr1/groups/12cr1.yaml]: You can define multiple expressions in the same command. I mean:
```yaml
- >-
  sed -i
  -e "s:HOSTNAME:${hostnamevar}:g" 
  -e "s:DOMAINNAME:`cat /etc/resolv.conf | grep search | sed 's/search //'`:g"
   /etc/motd
```
This speedup things I think.

**Comment by kurtz81** [comment]
add fix to compact the sed expression

File: envs/oracledb/12cr1/groups/12cr1.yaml (hunk 1 above)
Comment (kurtz81) [reply envs/oracledb/12cr1/groups/12cr1.yaml]: added fix to compact the sed expression

Comment (kurtz81) [reply envs/oracledb/12cr1/groups/12cr1.yaml]: updated section networks and profiles

**Pull request merged**